src/documents/ComponentConfigurationOptions.md

```diff
@@ -38,73 +36,72 @@ The following options are supported by all component grades:
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
 <code>fluid.defaults("component.name", {
-    nickName: "myComponentName",
+    invokers: {
+        inv1: {...},
+        inv2: {...},
+    },
     ...
 });</code>
 </pre></td>
   </tr>
   <tr>
     <th>See also</th>
-    <td><code><a href="https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/Fluid.js#L2100-L2107">fluid.computeNickName</a></code></td>
+    <td><a href="Invokers.md">Invokers</a></td>
   </tr>
 </table>
 
-### mergePolicy ###
+### `members` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object providing instructions for how particular options should be merged when integrator options are merged with default values.</td>
+    <td>A record defining properties to be added to the component object. These can be anything, including methods, strings, objects, etc. Definitions are evaluated as IoC expressions.</td>
   </tr>
   <tr>
     <th>Notes</th>
-    <td>It is uncommon to need this option. The most common use case is to protect "exotic values" derived from some external library or framework from being corrupted by the options merging/expansion process by use of the "nomerge" policy.</td>
+    <td>Defining a method as a Function in <code>members</code> will differ from <code>invokers</code> in that the arguments of members are not resolved at invocation time. The use of such function members is not recommended except where very high invocation performance is required.
+        The right-hand-side may contain an <a href="ExpansionOfComponentOptions.md">expander</a> definition, which may perhaps itself resolve onto an <a href="Invokers.md">invoker</a>.</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
 <code>fluid.defaults("component.name", {
-    mergePolicy: {
-        option1: "noexpand",
-        option2: "nomerge",
-        ....
+    members: {
+        member1: "{that}.options.optionsValue",
+        member2: "{theOther}.dom.otherSelector",
     },
     ...
 });</code>
 </pre></td>
   </tr>
-  <tr>
-    <th>See also</th>
-    <td><a href="OptionsMerging.md">Options Merging</a></td>
-  </tr>
 </table>
 
-### invokers ###
+### `events` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object defining methods on the component whose arguments are resolved from the environment as well as the direct argument list at invocation time.</td>
+    <td>A record containing key/value pairs that define the events the component will fire: the keys are the event names, the values define the type of the event (see <a href="InfusionEventSystem.md">Infusion Event System</a> for information on the different event types).</td>
+  </tr>
+  <tr>
+    <th>Notes</th>
+    <td>The Framework will create event firers for the listed events. It is the responsibility of the component to fire the events at the appropriate times.</td>
```

Comment:
Is it worth clarifying here that the primary component lifecycle events, such as <code>onCreate</code> and <code>onDestroy</code> are fired automatically by the framework?